### ISSUE TITLE
USSD Support

### ISSUE THREAD

**Issue by aseaudi** [herlesupreeth/docker_open5gs#285]
Hello @herlesupreeth 

Could you advise if I can test USSD using docker_open5gs ? and how ?

Thanks

**Comment by herlesupreeth** [comment]
Sadly, its not possible.

**Comment by aseaudi** [comment]
Why ? what is the limitation ?

**Comment by abouillot-stl** [comment]
USSD is not part of 4g/5g. It requires 2g/3g network services.

Envoyé à partir de Outlook pour Android<https://aka.ms/AAb9ysg>
________________________________

**Comment by aseaudi** [comment]
What about 3GPP TS 24.390 "Universal Mobile Telecommunications System (UMTS); LTE; Unstructured Supplementary Service Data (USSD)  using IP Multimedia (IM) Core Network (CN) subsystem IMS;" ?

**Comment by herlesupreeth** [comment]
> What about 3GPP TS 24.390 "Universal Mobile Telecommunications System (UMTS); LTE; Unstructured Supplementary Service Data (USSD) using IP Multimedia (IM) Core Network (CN) subsystem IMS;" ?

That requires a XCAP server and the one in kamailio does not support features needed for IMS

**Comment by aseaudi** [comment]
@herlesupreeth is USSD still not possible using docker_open5gs ?

**Comment by herlesupreeth** [comment]
No, its not supported. The state remains the same as I mentioned before
